packages/client/src/App.tsx

```diff
@@ -29,9 +25,19 @@ const cacheRtl = createCache({
 	// if you want to retain the auto-prefixing behavior.
 	stylisPlugins: [prefixer, stylisRTLPlugin],
 });
-//////////////////////////////////////////////////////////////////////////////
+const checkIsLoggedIn = () => {
+  if(window.localStorage.getItem("access_token")){
+    return true;
+  }else
+    return false;
+}
```

Comment:
`return Boolean(window.localStorage.getItem("access_token"));`
will be enough and one liner

Reply:
@marwank7